mlrun/model_monitoring/db/stores/sqldb/sql_store.py

```diff
@@ -539,8 +522,12 @@ def _create_tables_if_not_exist(self):
 
         for table in self._tables:
             # Create table if not exist. The `metadata` contains the `ModelEndpointsTable`
+            db_name = self._sql_connection_string.split("/")[-1]
```

Comment:
https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy.engine.make_url

I suggest the following more idiomatic parsing:

```py
from sqlalchemy.engine import make_url

db_name = make_url(self._sql_connection_string).database
```